game_event/tests.py

```diff
@@ -96,6 +97,22 @@ def test_create_game_event_valid_form(self, client, court, court_ball_game):
         client.post(url, data=form_data)
         assert GameEvent.objects.count() == count_of_game_events_before + 1
 
+    def test_player_added_after_creation_of_game_event(self, client, court, court_ball_game, player):
+        client.force_login(player.user)
+        count_events_of_player_before = GameEventPlayer.objects.filter(player=player).count()
+        url = reverse(process_game_event_form)
+        time_now_plus_one_day = (timezone.now() + timedelta(days=1)).strftime('%Y-%m-%dT%H:%M')
+        form_data = {
+            'time': time_now_plus_one_day,
+            'level_of_game': 5,
+            'min_number_of_players': 2,
+            'max_number_of_players': 4,
+            'court': court.courtID,
+            'ball_game': TEST_BALL_GAME,
+        }
+        client.post(url, data=form_data)
+        assert GameEventPlayer.objects.filter(player=player).count() == count_events_of_player_before + 1
```

Comment:
Let's also assert assert that the number of players for the event was increased by 1.